config/test/test.config.js

```diff
@@ -34,6 +34,10 @@ module.exports = {
         "jest-extended/all",
         "<rootDir>/config/test/test-setup.js",
         "<rootDir>/config/test/custom-matchers.js",
+
+        // math-input uses these matchers. We can remove this once we
+        // transition these tests to RTL
+        "<rootDir>/node_modules/jest-enzyme/lib/index.js",
```

Comment:
Can you file a JIRA ticket for this?

Reply:
Done: https://khanacademy.atlassian.net/browse/LP-11633 (I've also added it to this TODO comment)